cmake/modules/TTMLIRBuildTypes.cmake

```diff
@@ -1,9 +1,9 @@
 # Asan build type
 set(CMAKE_C_FLAGS_ASAN
-    "${CMAKE_C_FLAGS_DEBUG} -fsanitize=address -fno-omit-frame-pointer" CACHE STRING
+    "${CMAKE_C_FLAGS_DEBUG} -fsanitize=address -fno-omit-frame-pointer -g" CACHE STRING
```

Comment:
Why `-g`. It should be added with debug build right?

Reply:
I don't know, this was introduced in https://github.com/tenstorrent/tt-mlir/pull/1562, I've just rebased this branch on top of it to build it with ASAN.

Reply:
my bad. I have removed `-g` in #1562 . Thanks for pointing it out.